.clang-tidy

```diff
@@ -0,0 +1,28 @@
+---
+Checks:          '*'
+WarningsAsErrors: '*'
```

Comment:
Added this. However I can't seem to get `run-clang-tidy.py` to return a non-zero error code. So this basically has no impact on the return code, just an minor impact on the stdout from clang-tidy.